main.js

```diff
@@ -74,6 +75,9 @@ app.on('ready', function () {
   });
   // ----- Google auth -----
 
+  ipcMain.on('start-download', (event, items, access_token) => {
```

Comment:
Can we define constants for these IPC message names to avoid magic strings in the code?